01-shapefile-attributes.Rmd

```diff
@@ -513,25 +509,26 @@ legend("bottomright",
 
 
 <div id="challenge" markdown="1">
-##Challenge
+##Challenge: Plot Color by Attribute
```

Comment:
@mjones01 please change to "Plot Polygon by Attribute" 
plot color by attribute doesn't make sense.